chainer/functions/array/vstack.py

```diff
@@ -52,10 +52,11 @@ def vstack(xs):
         xs (list of :class:`~chainer.Variable` or :class:`numpy.ndarray` or \
         :class:`cupy.ndarray`):
             Input variables to be concatenated. The variables must have the
-            same ``ndim``. Also, when the variables have vertical dimension
-            (i.e. :math:`ndim \\geq 1`), the variables must have the same
-            shape, except in the vertical (row wise) dimension.
-
+            same ``ndim``. When the variables have the second axis (i.e.
```

Comment:
Ok, maybe I understand. When using 1dim array, the array is column-wide (or 2nd axis wide) and the size of vertical dimension(or 1st axis) always 1.